gammapy/maps/axes.py

```diff
@@ -45,6 +45,18 @@ def pix_to_coord(edges, pix, interp="lin"):
     return scale.inverse(interp_fn(pix))
 
 
+PLOT_AXIS_LABEL = {
+    "energy": "Energy",
+    "energy_true": r"E$_{\rm true}$",
+    "offset": "Offset",
+    "rad": r"$\delta p$",
```

Comment:
I don't know is $\delta p$ is clearer than `rad`

Reply:
I took this definition from [the IRF axis ](https://docs.gammapy.org/0.19/irf/index.html#irf-axis-naming). Effectively $\delta p$ is not really clear to all users, while `rad` is more common.